flake.nix

```diff
@@ -11,7 +11,8 @@
       }) final;
 
       mlir = final.llzk_llvmPackages.mlir;
-
+      libllvm = final.llzk_llvmPackages.libllvm;
+      llvm = final.llzk_llvmPackages.llvm;
```

Comment:
I don't think these should go into the overlay since they will cause a lot of confusion.